tests/tests_change_indication_on_repeat_run.yml

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# This file was generated by ensure_provider_tests.py
+---
+- hosts: all
+  vars:
+    interface: testnic1
+    type: veth
+  name: Test change indication on repeat run
+  tasks:
+    - include_tasks: tasks/manage_test_interface.yml
+      vars:
+        state: present
+    - include_tasks: tasks/assert_device_present.yml
+    - block:
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                state: up
+                type: ethernet
+                ip:
+                  dhcp4: "no"
+                  auto6: "no"
+          register: __network_connections_result
+        - name: Assert change:true
+          assert:
+            that: __network_connections_result.changed
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                state: up
+                type: ethernet
+                ip:
+                  dhcp4: "no"
+                  auto6: "no"
```

Comment:
This variable can be set in line 5 for the playbook then it does not need to be repeated and it would ensure that the profiles are always the same.

Reply:
I had trouble on this. Could you provide an example?

Reply:
> This variable

Which variable?

> I had trouble on this.

What did you try, and what was the result?

Reply:
I tried to place the `network_connections` at the host level variable, and then remove that from `import_role` task. Not work.
Even with this in `import_role` task will not work:

```yaml
    - block:
        - import_role:
            name: linux-system-roles.network
          vars:
            - network_connections: " {{ network_connections }} "
          register: __network_connections_result
```

Reply:
not sure what the problem is - here is an example playbook calling a role with a variable defined at the host level - https://gist.github.com/richm/e9d338d8b39254b84b90fd2bf0b0b0a1

Reply:
Thanks for the suggestion. The code is working now.